.changeset/healthy-plants-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+"chainlink": patch
+---
+
+#removed history depth to finality depth validation
```

Comment:
This is not a NOP facing change, so maybe use #internal instead

Reply:
Done. Thanks